addons/website/models/res_config_settings.py

```diff
@@ -19,15 +19,11 @@ def _default_website(self):
     website_country_group_ids = fields.Many2many(related='website_id.country_group_ids', readonly=False)
     website_company_id = fields.Many2one(related='website_id.company_id', string='Website Company', readonly=False)
     website_logo = fields.Binary(related='website_id.logo', readonly=False)
-    language_ids = fields.Many2many(related='website_id.language_ids', relation='res.lang',
-        readonly=False)
+    language_ids = fields.Many2many(related='website_id.language_ids', relation='res.lang', readonly=False)
     website_language_count = fields.Integer(string='Number of languages', compute='_compute_website_language_count', readonly=True)
-    website_default_lang_id = fields.Many2one(
-        string='Default language', related='website_id.default_lang_id', readonly=False,
-        relation='res.lang')
-    website_default_lang_code = fields.Char(
-        'Default language code', related='website_id.default_lang_code', readonly=False,
-        )
+    website_default_lang_id = fields.Many2one(string='Default language', related='website_id.default_lang_id',
+                                              readonly=False, relation='res.lang')
+    website_default_lang_code = fields.Char('Default language code', related='website_id.default_lang_id.code', readonly=False)
```

Comment:
open question, does it makes sense, performance-wise to store it? as `default_lang_code` was stored

Reply:
If you are talking about this particular field (`website_default_lang_code`), it is a res.settings field so transient so can't store.

But if you were talking about  `default_lang_code` directly, it should not change anything.

I did a quick check to ensure I did not break request numbers with this PR.

Result:

```
|------------|-BEFORE-|-NOW-|
| COLD       |        |     |
|   PUBLIC   |        |     |
|     EN     |    455 | 459 |
|     FR     |    465 | 467 |
|   ADMIN    |        |     |
|     EN     |    688 | 691 |
|     FR     |    725 | 727 |
| HOT        |        |     |
|   PUBLIC   |        |     |
|     EN     |     41 |  43 |
|     FR     |     45 |  48 |
|   ADMIN    |        |     |
|     EN     |     63 |  65 |
|     FR     |     67 |  69 |
-----------------------------
```